package day11;

public class ExceptionExam1 {

    public static void main(String[] args) {
        int[] arr = {1, 2, 3, 4};
        System.out.println(arr[4]); // 예외 발생

        System.out.println("해야할 일1");
        System.out.println("해야할 일2");
        System.out.println("해야할 일3");
        System.out.println("종료");
    }
}
